examples/reference/panes/Perspective.ipynb

```diff
@@ -17,82 +21,250 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The ``Perspective`` pane provides an interactive visualization component for large, real-time datasets built on the [Perspective](https://perspective.finos.org/) project.\n",
+    "The ``Perspective`` pane provides a powerful visualization component for large, real-time datasets built on the [Perspective Project](https://perspective.finos.org/). **`Perspective` brings *excel-like* capabilities to your data app**. Check out the [Perspective Examples Gallery](https://perspective.finos.org/examples/) for inspiration.\n",
+    "\n",
+    "The ``Perspective`` pane is a very good alternative to the [`Tabulator`](../widgets/Tabulator.ipynb) widget.\n",
     "\n",
-    "#### Parameters:\n",
+    "## Parameters:\n",
     "\n",
     "For details on other options for customizing the component see the [layout](../../how_to/layout/index.md) and [styling](../../how_to/styling/index.md) how-to guides.\n",
     "\n",
     "* **``aggregates``** (dict): Aggregation spec, e.g. {x: \"distinct count\"}\n",
-    "* **``columns``** (list): List of displayed columns.\n",
+    "* **``columns``** (list or dict): List of column names to display or a dictionery with column configuration options.\n",
```

Comment:
Dicts works. But I don't understand why because the underlying parameter is a `List`.